Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -407,6 +416,19 @@ YUI.add('ez-languageselectionboxview', function (Y) {
              */
             selectedBaseLanguageCode: {
                 value: null
+            },
+
+            /**
+             * Defines what the list o translations will contain.
+             * If it is set to TRUE then list of languages will contain not existing (new) translations.
+             * Otherwise list of language will contain already existing translations.
+             *
+             * @attribute baseTranslation
+             * @type {Boolean}
+             * @default false
+             */
+            newTranslationsOnList: {
```

Comment:
the name of this attribute is bad, it describes the technical behaviour, not the actual feature.

Actually, the language selection box should have 2 modes:

1. a "normal" mode (default) where we let the user chooses amongst the languages in `existingTranslations` (this attribute should be renamed, see below).
2. a translation mode where we provide the non existing languages

To make things a bit clearer, some attributes should be renamed:
* `existingTranslations` => `referenceLanguageList`
* `availableTranslations` (which is not really defined and documented btw) => `systemLanguageList`
* `newTranslationsOnList` => `translationMode` (false by default).

Some comments should also be updated to use *language* (or *language code*) instead of *translation* when the involved code is applicable in *normal* mode and to remove any reference to *content* since the language selection box will be used for others things than content.

Last detail, in the template and the CSS, you can not use `ez-languageselectionbox-newtranslations` and `ez-new-translation` classes since we are not always displaying potential new translations there.